src/cfml/system/services/ServerService.cfc

```diff
@@ -681,9 +767,21 @@ component accessors="true" singleton {
 	*/
 	function saveServerJSON( required string directory, required struct data ) {
 		var filePath = arguments.directory & '/server.json';
+		if(fileExists(filePath)) {
+		  var existingProps = readServerJSON(directory);
+		  if(!isNull(data.name) && (isNull(existingProps.name) || existingProps.name != data.name)) {
+		    filePath = arguments.directory & '/server-#data.name#.json';
```

Comment:
What Brad said.  Basically it allows more than one server config per project.  I might want to test my codebase with both Lucee and Adobe, and maybe even at once-- versus setting an engine, starting, stopping, changing the engine, starting, stopping, back and forth.